src/app/pages/administration/jobs/system-triggers-view.jsx

```diff
@@ -41,84 +45,139 @@ const useStyles = makeStyles({
 });
 
 function ChplSystemTriggersView(props) {
+  const { dispatch } = props;
   const [triggers, setTriggers] = useState([]);
+  const [isConfirming, setIsConfirming] = useState(false);
+  const [pendingAction, setPendingAction] = useState({});
+  const [pendingMessage, setPendingMessage] = useState('');
   const classes = useStyles();
 
+  const confirmDelete = (item) => {
+    setIsConfirming(true);
+    setPendingAction({
+      action: 'delete',
+      payload: {
+        name: item.triggerName,
+        group: item.triggerGroup,
```

Comment:
Recommend using the spread syntax here, as that's the syntax you're using in the other payload object on the user triggers job. The alternative could be to switch the other one to be explicit, but I've had that come back to bite me in the past when the object structure changed, or an element I needed hadn't been explicitly included
```suggestion
   ...item,
```